proposals/16-first-filecoin-storage-deal.md

```diff
@@ -0,0 +1,130 @@
+# Proposal: Reduce friction and time to first verified deal for new Filecoin storage clients
```

Comment:
If I understand this proposal correctly the scope is basic filecoin deals of the type Andrew and Jonathan's walkthroughs attempted to make, not deals related to the ongoing FIL+ (i.e. verified clients/verified deals) effort.  If this is right then the term "verified deal" unfortunately has some extra baggage you probably didn't intend to bring along.  Possible wording changes: "first completed deal", "first finalized deal"

Reply:
Thanks for calling that out, @ZenGround0. You're right that it wasn't my intention. I just remember that the process for getting a success message is somewhat long, so the goal is getting to the stage at which the user is confident their data is being stored by a miner.